### PR TITLE
[doc test] tests hashes of external examples

### PR DESCRIPTION
**Pull request by aslonnie**
by asking people to update the digest of the files as a confirmation.

### CODE REVIEW

**Comment by aslonnie** [comment]
follow up to the removal of the github action checker.

File: doc/BUILD [hunk 1]
```diff
@@ -1,6 +1,8 @@
 load("//bazel:python.bzl", "py_test_run_all_subdirectory")
 load("//bazel:python.bzl", "py_test_run_all_notebooks")
 load("//bazel:python.bzl", "doctest")
+load("@py_deps_buildkite//:requirements.bzl", ci_require = "requirement")
```
Comment (aslonnie) [inline doc/BUILD:4]: let me get the buildifier format into another PR.

Comment (aslonnie) [reply doc/BUILD:4]: here it goes: https://github.com/ray-project/ray/pull/46458

**Comment by aslonnie** [comment]
ping @matthewdeng , here is your follow up.

File: doc/BUILD [hunk 2]
```diff
@@ -351,7 +353,7 @@ py_test(
 py_test_run_all_subdirectory(
     size = "enormous",
     include = ["external/*.py"],
-    exclude = [],
+    exclude = ["external/test_hashes.py"],
```
Comment (richo-anyscale) [inline doc/BUILD:356]: Is there a reason to put it here when it will then need to be explicitly ignored?

Comment (aslonnie) [reply doc/BUILD:356]: it is related to files in that directory?

supposedly files in that directory are external examples, and this file is a signing sheet for changes.

**Review by richo-anyscale — APPROVED** [review]

**Pull request merged**